client/my-sites/stats/all-time/index.jsx

```diff
@@ -97,27 +106,35 @@ module.exports = React.createClass( {
 						<p>{ this.translate( 'These are your site\'s overall total number of Posts, Views and Visitors as well as the day when you had the most number of Views.' ) }</p>
 					</div>
 
-				<ul className="stats-all-time__list">
-					<li className="stats-all-time__list-item stats-all-time__item-posts">
-						<Gridicon icon="posts" size={ 18 } />
-						<span className="stats-all-time__label">{ this.translate( 'Posts' ) }</span>
-						<span className={ classNames( valueClass, { 'is-low': this.isLow( this.props.allTimeList.response.posts ) } ) }>{ this.ensureValue( this.props.allTimeList.response.posts ) }</span>
+				<ul className="module-tabs">
+					<li className={ classSets.posts }>
+						<span className="no-link">
+							<Gridicon icon="posts" size={ 18 } />
+							<span className="label">{ this.translate( 'Posts' ) }</span>
+							<span className={ classNames( valueClass, { 'is-low': this.isLow( this.props.allTimeList.response.posts ) } ) }>{ this.ensureValue( this.props.allTimeList.response.posts ) }</span>
```

Comment:
I see that this existed already before you touch it, but these lines are so long and could be shorted with the help of a helper variable, making them hopefully easier to read

```js
var allTimeList = this.props.allTimeList.response;

'is-low': this.isLow( allTimeList.posts )
```